index.d.ts

```diff
@@ -27,6 +27,7 @@ declare module 'prismarine-auth' {
     }>
     // Returns a Minecraft Java Edition auth token
     getMinecraftJavaToken(options?: {
+      fetchCertificate?: boolean,
```

Comment:
is this a certificate or many (plural)?

Reply:
it's a keypair, the endpoint is called (Player) Certificates
<https://wiki.vg/Mojang_API#Player_Certificates>

Reply:
`fetchKeypair` instead then ?

Reply:
i'd rename it to `fetchCertificates` or `fetchPlayerCertificates` as per the endpoint because it also returns one or two signatures from mojang of the public key so it's not just the keypair

Reply:
Done